kdl_parser/package.xml

```diff
@@ -22,16 +22,24 @@
 
   <buildtool_depend version_gte="0.5.68">catkin</buildtool_depend>
 
+  <build_depend>cmake_modules</build_depend>
+  <build_depend>liburdfdom-headers-dev</build_depend>
   <build_depend version_gte="1.3.0">orocos_kdl</build_depend>
   <build_depend>rosconsole</build_depend>
-  <build_depend>roscpp</build_depend>
+  <build_depend>tinyxml</build_depend>
   <build_depend>urdf</build_depend>
-  <build_depend>cmake_modules</build_depend>
-  <build_depend>rostest</build_depend>
 
-  <run_depend version_gte="1.3.0">orocos_kdl</run_depend>
-  <run_depend>rosconsole</run_depend>
-  <run_depend>roscpp</run_depend>
-  <run_depend>urdf</run_depend>
+  <build_export_depend>liburdfdom-headers-dev</build_export_depend>
+  <build_export_depend>orocos_kdl</build_export_depend>
```

Comment:
I think that this should have a minimum version like the other references to `orocos_kdl` : `version_gte="1.3.0"`

Also as this is a build, build_export and exec_depend it could be collapsed into a single `depend` tag (same for tinyxml and urdf below)

Reply:
Fixed in 01f88ca